src/routes/api/sms/+server.ts

```diff
@@ -0,0 +1,161 @@
+import { error, json } from '@sveltejs/kit';
+import { safeParse } from 'valibot';
+
+import { CategoryMessage, collectorSchema, TypeMessage } from '$lib/schemas/collector';
+import { smsSchema } from '$lib/schemas/sms';
+
+import { db } from '$server/db';
+import { type Message, messages } from '$server/db/schema/messages';
+import { responses } from '$server/db/schema/responses';
+
+import type { RequestHandler } from './$types';
+
+/**
+ * Determine if the message is related to a previous message
+ * @param args
+ * @returns
+ */
+async function reladedToAMessage(args: { patientId: string }): Promise<Message[]> {
```

Comment:
```suggestion
async function isRelatedToMessage(args: { patientId: string }): Promise<Message[]> {
```

Mettre dans un dossier utils